mqtt.kafka.asyncapi.proxy/docker/compose/setup.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -e
+
+if [[ -z "$KAFKA_HOST" && -z "$KAFKA_PORT" ]]; then
+  export KAFKA_HOST=host.docker.internal
+  export KAFKA_PORT=9092
+  echo "==== This example requires env vars KAFKA_HOST and KAFKA_PORT for a running kafka instance. Setting to the default ($KAFKA_HOST:$KAFKA_PORT) ===="
+fi
+
+NAMESPACE=zilla-mqtt-kafka-asyncapi-proxy
+
+# Start or restart Zilla
+if [[ -z $(docker-compose -p $NAMESPACE ps -q zilla) ]]; then
+  docker-compose -p $NAMESPACE up -d
+
+  # Create the mqtt topics in Kafka
+  docker run --rm bitnami/kafka:3.2 bash -c "
```

Comment:
```suggestion
  docker run --rm bitnami/kafka bash -c "
```
It is best to just use `latest` for an init container